src/core/qgsmessagelog.h

```diff
@@ -140,6 +140,9 @@ class CORE_EXPORT QgsMessageLogConsole : public QObject
   public:
     QgsMessageLogConsole();
 
+  protected:
+    QString formatLogMessage( const QString &message, const QString &tag, Qgis::MessageLevel level ) const;
```

Comment:
Perhaps you can to add a default INFO to Qgis::MessageLevel level

Reply:
Good call. Done.